admin/failures/overview/view.html

```diff
@@ -1,385 +1,81 @@
-<!DOCTYPE html>
-<html>
-    <head>
-        <meta charset="utf8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>Mærkelex Salg</title>
-        <style type="text/css">
-            @font-face { font-family: "Open Sans"; src: url("/assets/font/OpenSans-Regular.ttf") format("truetype"); }
-            @font-face { font-family: "Open Sans Light"; src: url("/assets/font/OpenSans-Light.ttf") format("truetype"); }
+<div class="container">
+    <table>
+        <thead> 
+            <tr>
+                <th title="Oprettelsesdato for ordren">Seneste fejl</th>
+                <th>Beskrivelse</th>
+                <th></th>
+            </tr>
+        </thead>
+        <tbody>
+            {{ # failedOrders }}
+            <tr class="unaccounted-order-row" data-id="{{ id }}">
```

Comment:
Trick: to make big changes like this render nicely in Github or other Git renderings, it sometimes pays off to split the change into two commits:

1. Remove the code no longer needed
2. Fix indentation

That would give me a nice view of what has been changed, and a separate view of indentation fixes. Right now, I have no chance of reviewing what parts you have actually removed, because I cannot see what has been kept!